File: solutions/easy/Convert Binary Number in a Linked List to Integer.java
// Title: Convert Binary Number in a Linked List to Integer
            // Difficulty: Easy
            // Language: Java
            // Link: https://leetcode.com/problems/convert-binary-number-in-a-linked-list-to-integer/

        int num = 0;
        ListNode temp = head;
        while (temp != null) {
            num = (num << 1) | temp.val;
            temp = temp.next;
        }
        return num;
    }
}

    public int getDecimalValue(ListNode head) {
class Solution {

File: solutions/easy/Convert Binary Number in a Linked List to Integer.c
// Title: Convert Binary Number in a Linked List to Integer
            // Difficulty: Easy
            // Language: C
            // Link: https://leetcode.com/problems/convert-binary-number-in-a-linked-list-to-integer/

 * };
 */
int getDecimalValue(struct ListNode* head) {
     int num = 0;
       struct  ListNode * temp = head;
        while (temp != NULL) {
            num = (num << 1) | temp->val;
            temp = temp->next;
        }
        return num;
}
